### ISSUE TITLE
Post-login automation does not send my command, merely types it in, for Windows

### ISSUE THREAD

**Issue by Firsh** [connectbot/connectbot#1128]
The server is Windows OpenSSH. Anything, even just
`dir`
put into post-login automation just gets copy pasted into the terminal but Enter is not pressed for me. It works on Ubuntu servers. Is there a special way of ensuring the command gets sent, or is this a bug? I tried newlines and ; character.

**Comment by skingzd** [comment]
me too.Use it connect to windows,always need to press enter manually .

**Comment by nolddor** [comment]
Same here connecting via SSH to Debian server.

@kruton any chance to look into this?

**Comment by ckujau** [comment]
Duplicate of #1097? Although I've experienced the same for years now, not sure why both these issues were only opened recently.

**Comment by atikattar1104** [comment]
Same For Termux As Well

**Comment by lordofthemoon** [comment]
I've just started using ConnectBot to do this, and I see the same thing.  I'm connecting to a Raspberry Pi running raspbian stretch, using a keypair so that I don't have to use a password.  I just want to execute a single command on the remote box and then disconnect, but the command in "post-login automation" just gets placed on the command line, and I have to press enter to actually execute it. (v1.9.9 according to menu > Help).

**Comment by Alteus** [comment]
Two years later, and this issue is still open. As the matter of fact, the solution is trivial, just hit enter key after your command in the post-login automation box.

**Comment by quentinkrammer** [comment]
I had the same issue on a debian server. I solved it by appending a newline character to the end of the shell command.

Example:
All I want is for my Debian server to shutdown as soon as I connect via SSH.

This Post Login-Automation script would require me to manually press Enter after connecting: 
`sudo poweroff`

Whereas this script gets immediately executed: 
```
sudo poweroff

```

**Comment by Firsh** [comment]
Yes, new line (an empty line at the end in the textbox) works fine on Windows SSH as well as Ubuntu now.

**Comment by atikattar1104** [comment]
Maybe This Issue Can Be Closed Now.

**Comment by Roberto-Gentili** [comment]
> Yes, new line (an empty line at the end in the textbox) works fine on Windows SSH as well as Ubuntu now.

Not working for me on Windows SSH

**Comment by atikattar1104** [comment]
Try Adding 2 New Lines. Sometimes, Only One New Line Can Be Considered As False Positive By The Systems.

**Comment by Roberto-Gentili** [comment]
I had already tried with more lines but it didn't work
